### PR TITLE
Convert to number format for table of contents

### PR DESCRIPTION
**Pull request by hauchongtang**
Refer to #255 

## Changes
* Better formatting with numbering (Need deploy and see if there's any bugs first)
* Add Before and After screenshots for Inc / Decrement portion of the UG
* Add before and after screenshots for one of the list commands since there was some confusion from testers: `listSuppliers`
* Include definitions of GUI / CLI / Colors in the appendix

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -385,3 +410,28 @@ Clears and deletes all suppliers in Salesy.
...
+**Detailed Explanation**
+
+| Color          | Meaning                                                      | How is it determined ?                                                                              |
+|----------------|--------------------------------------------------------------|-----------------------------------------------------------------------------------------------------|
+| 🔴&nbsp;Red    | Running `low`, restock as soon as possible.                  | **Less than 120%** of minimum stock specified.                                                      |
+| 🟠&nbsp;Orange | Running at a `moderate` level, can consider restocking soon. | **More than and equal to 120%** of minimum stock specified and **less than 165%** of minimum stock. |
+| 🟢&nbsp;Green  | Running at a `healthy` level, no worries about restocking.   | **More than and equal to 165%** of minimum stock specified.                                         |
+
+[//]: # (@@author hauchongtang-reused)
+[//]: # (Both definitions: 6.2 and 6.3 taken from https://byjus.com/gate/difference-between-cli-and-gui/#:~:text=GUI%20lets%20a%20user%20interact,offer%20both%20CLI%20and%20GUI.)
```
Comment (hauchongtang) [inline docs/UserGuide.md:427]: This is the format for commenting in markdown (won't render on the webpage)

File: docs/UserGuide.md [hunk 2]
```diff
@@ -3,7 +3,10 @@ layout: page
 title: Salesy User Guide v1.3
 ---
 
-Salesy is an all-in-one tool for NUS canteen vendors who want to keep track of their tasks, inventory and suppliers, optimized for use via a Command Line Interface (CLI) while still having the benefits of a Graphical User Interface (GUI). With Salesy, you can expect a good visualization of your inventory and increased efficiency with your bookkeeping routines.
+Salesy is an all-in-one tool for NUS canteen vendors who want to keep track of their tasks, inventory and suppliers. 
+With Salesy, you can expect a good visualization of your inventory and increased efficiency with your bookkeeping routines.
```
Comment (weesiau) [inline docs/UserGuide.md:7]:
```suggestion
With Salesy, you can expect a good visualization of your inventory and increased efficiency of your bookkeeping routines.
```

File: docs/UserGuide.md [hunk 3]
```diff
@@ -3,7 +3,10 @@ layout: page
...
+Salesy optimized for use via a [Command Line Interface](#62-command-line-interface-cli) (CLI) while still having the benefits of
```
Comment (weesiau) [inline docs/UserGuide.md:8]:
```suggestion
Salesy is optimized for use via a [Command Line Interface](#62-command-line-interface-cli) (CLI) while still having the benefits of 
```

**Review by weesiau — APPROVED** [review]
LGTM, good effort on the heading numbers

**Pull request merged**